x-pack/plugins/data_enhanced/server/search/session/session_service.test.ts

```diff
@@ -4,19 +4,88 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { of } from 'rxjs';
+import { BehaviorSubject, of } from 'rxjs';
 import type { SavedObject, SavedObjectsClientContract } from 'kibana/server';
 import type { SearchStrategyDependencies } from '../../../../../../src/plugins/data/server';
 import { savedObjectsClientMock } from '../../../../../../src/core/server/mocks';
 import { BackgroundSessionStatus } from '../../../common';
 import { BACKGROUND_SESSION_TYPE } from '../../saved_objects';
-import { BackgroundSessionDependencies, BackgroundSessionService } from './session_service';
+import {
+  BackgroundSessionDependencies,
+  BackgroundSessionService,
+  INMEM_TRACKING_INTERVAL,
+  MAX_UPDATE_RETRIES,
+  SessionInfo,
+} from './session_service';
 import { createRequestHash } from './utils';
+import moment from 'moment';
```

Comment:
We already use moment in the data plugin, so I don't think it should matter. :)

Reply:
I'd have to use this kind of syntax: `new Date(new Date().getTime() - (2*60*1000));` rather than `moment().subtract(2, 'm')`.
I'm really not a fan, but if you insist :shrug:

Reply:
nit a bit shorter: new Date(Date.now() - (2*60*1000));

One q: are there timezones risks in play? Maybe movement handles something in addition related to tz?